pipelines/test/kwok/cluster.yaml

```diff
@@ -45,3 +51,11 @@ pipeline:
 
       kubectl wait --for=condition=Ready nodes --all
       kubectl cluster-info
+      export KUBERNETES_SERVICE_HOST="127.0.0.1"
+      export KUBERNETES_SERVICE_PORT="32764"
```

Comment:
These will be set only for the current pipeline step, which is only this shell process and children that run kwok.
Something like this will fail:
```
...
  - uses: test/kwok/cluster
  - runs: |
      set -u
      echo $KUBERNETES_SERVICE_HOST
```

with
```
...
2025/04/15 10:56:53 WARN /bin/sh: KUBERNETES_SERVICE_HOST: parameter not set
```